caddyfile/dispenser_test.go

```diff
@@ -198,6 +198,47 @@ func TestDispenser_NextBlock_MakesProgressOrStopsOnEOF(t *testing.T) {
 	t.Fatalf("NextBlock(): exceeded %d iterations without terminating", maxIters)
 }
 
+func TestDispenser_NextBlock_MultipleLevelsOfNestedBlocks(t *testing.T) {
+	input := `foobar1 {
+			  	sub1 arg1
+			  	sub2
+			  	sub_foobar1 sub_foobar1_arg {
```

Comment:
These three lines have mixed whitespace (spaces, tabs). Could you please align it to spaces?